src/data_model/eng2math.jl

```diff
@@ -469,6 +469,11 @@ function _map_eng2math_transformer!(data_math::Dict{String,<:Any}, data_eng::Dic
 
                 data_math["transformer"]["$(transformer_2wa_obj["index"])"] = transformer_2wa_obj
 
+                # add regcontrol items to math model
+                if haskey(eng_obj,"controls") && eng_obj["bus"][2]==eng_obj["bus"][w]
+                    data_math["transformer"]["$(transformer_2wa_obj["index"])"]["controls"] = eng_obj["controls"]
+                    data_math["transformer"]["$(transformer_2wa_obj["index"])"]["tm_fix"] = fill(false,length(transformer_2wa_obj["f_connections"]))
+                end
```

Comment:
does this always mean the second winding is the one being controlled, and all phases are controlled, even in cases where there might not be controls for each phase (e.g., in cases where transformers are banks, and a regcontrol doesn't exist for each object in the bank)?